.github/files/multi-platform.Dockerfile

```
@@ -1,7 +1,11 @@
-FROM alpine
+FROM alpine:3.21
+
 ARG TARGETPLATFORM
 
 LABEL org.opencontainers.image.description="Blazing fast and lightweight tile server with PostGIS, MBTiles, and PMTiles support"
+LABEL org.opencontainers.image.source="https://github.com/maplibre/martin"
+LABEL org.opencontainers.image.licenses="Apache-2.0 OR MIT"
+
```

We can actually add a bit more static metadata:

```suggestion
LABEL org.opencontainers.image.documentation="https://maplibre.org/martin/"
LABEL org.opencontainers.image.vendor="maplibre"

```